docs/user-guide/common-use-cases/shared-wallets.md

```diff
@@ -0,0 +1,450 @@
+  },
+  "template": {
+    "all": [
+      "cosigner#0",
+      "cosigner#1"
+    ]
+  }
+}
+```
+
+### Spending transaction
+
+Our shared wallets are fully operational now. In particular we have access to the addresses of the wallets via [`GET /shared-wallets/{walletId}/addresses`](https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/listSharedAddresses) endpoint. We can get the address and fund it from the [faucet](https://testnets.cardano.org/en/testnets/cardano/tools/faucet/) so that we can spend from our wallet later.
+
+After I have funded my "Cosigner#0 wallet" I see that the balance changed on "Cosigner#1 wallet" as well. Both wallets have the same balance:
+
+```
+$ curl -X GET http://localhost:8090/v2/shared-wallets/2a0ebd0cceab2161765badf2e389b26e0961de2f | jq .balance
+
+{
+  "available": {
+    "quantity": 10000000000,
+    "unit": "lovelace"
+  },
+  "reward": {
+    "quantity": 0,
+    "unit": "lovelace"
+  },
+  "total": {
+    "quantity": 10000000000,
+    "unit": "lovelace"
+  }
+}
+
+$ curl -X GET http://localhost:8090/v2/shared-wallets/5e46668c320bb4568dd25551e0c33b0539668aa8 | jq .balance
+
+{
+  "available": {
+    "quantity": 10000000000,
+    "unit": "lovelace"
+  },
+  "reward": {
+    "quantity": 0,
+    "unit": "lovelace"
+  },
+  "total": {
+    "quantity": 10000000000,
+    "unit": "lovelace"
+  }
+}
+```
+
+Of course this is expected. Both co-owners have full knowledge of the balance, however they cannot spend it on their own. As required by the payment template the wallet needs both co-owners' signatures to spend from the wallet.
+
+Let's make a simple transaction. "Cosigner#0 wallet" owner will construct and sign the transaction on their end and then provide CBOR of this transaction to "Cosigner#1 wallet" owner. Then the Cosigner#1 can sign it on his side and submit it to the network.
+
+We will be using shared wallet's transaction enpoints:
+ - [`POST /shared-wallets/{walletId}/transactions-construct`](https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/constructSharedTransaction)
+ - [`POST /shared-wallets/{walletId}/transactions-sign`](https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/signSharedTransaction)
+ - [`POST /shared-wallets/{walletId}/transactions-submit`](https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/submitSharedTransaction)
+
+> :information_source: Note that at any point in the process of constructing and signing a transaction both "Cosigner#0" and "Cosigner#1" may decode the CBOR of transaction using [`POST /shared-wallets/{walletId}/transactions-decode`](https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/decodeSharedTransaction) endpoint and get all its details including, but not limited to, `inputs`, `outputs`, `fees`, `deposits`, `metadata` or `witness_count`.
```

Comment:
```suggestion
> :information_source: Note that at any point during the process of constructing and signing a transaction, both "Cosigner#0" and "Cosigner#1" may decode the transaction to its CBOR representation using the [`POST /shared-wallets/{walletId}/transactions-decode`](https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/decodeSharedTransaction) endpoint. Decoding the transaction makes it possible to see all details associated with the transaction, such as its `inputs`, `outputs`, `fees`, `deposits`, `metadata`, or `witness_count`.
```

Reply:
@jonathanknowles I think it should be rather: "both "Cosigner#0" and "Cosigner#1" may decode the transaction **from** its CBOR representation"?